### DISCUSSION TITLE
Random Query Parameter

### DISCUSSION THREAD

**Discussion by dannycoulombe** [directus/directus#6319] [Feature Requests]
It would be nice to be able to fetch data randomly. For instance :
/items/posts?fields=*&limit=1&random=1

**Comment by philleepflorence** [comment]
A possible solution, using the existing [API Parameter: Offset](https://docs.directus.io/reference/api/query/#offset)

```shell
# rand = random(n items)

/items/posts?fields=*&offset=rand&limit=1
```

**Reply by rijkvanzanten** [reply]
To clarify, the above is a suggestion, not how it currently works.

**Reply by philleepflorence** [reply]
Thanks...I will update my response.

**Comment by benhaynes** [comment]
Duplicate: https://github.com/directus/directus/discussions/3240
